lib/core.ex

```diff
@@ -7,8 +7,18 @@ defmodule Core do
 
   def create_playlist(channel_id) do
     video = Youtube.get_channel_last_video(channel_id)
-    Line.send_text_message("A new video was published #{video.video_title}")
-    create_playlist_from_video(video.video_id)
+
+    case video do
```

Comment:
I don't think the variable name `video` is accurate, the good thing is that with pattern-matching you often start naming within the pattern:
```suggestion
    case Youtube.get_channel_last_video(channel_id) do
```